goon/code/datums/browserOutput.dm

```diff
@@ -13,7 +13,10 @@ var/list/chatResources = list(
 	"goon/browserassets/css/browserOutput.css",
 	"goon/browserassets/css/browserOutput-dark.css",
 	"goon/browserassets/json/unicode_9_annotations.json",
-	"goon/browserassets/html/saveInstructions.html"
+	"goon/browserassets/html/saveInstructions.html",
+	"goon/newbrowserassets/dist/app.js",
+	"goon/newbrowserassets/dist/manifest.js",
+	"goon/newbrowserassets/dist/vendor.js",
```

Comment:
These shouldnt be under the `goon` directory since its not a goon asset, and that directory is there due to different licensing.

Reply:
There's quite a bit of code that is copied/translated from jquery to vanilla JS, so I'm thinking that this new code probably falls under the original goonchat license. Is it that goonchat uses GPL rather than AGPL?